src/js/utils/MesosStateUtil.js

```diff
@@ -110,14 +110,15 @@ const MesosStateUtil = {
     }, {});
   },
 
-  getTasksFromVirtualNetworkName({ tasks = [] } = {}, overlayName) {
+  getRunningTasksFromVirtualNetworkName({ tasks = [] } = {}, overlayName) {
     return tasks.filter(function(task) {
       const appPath = "container.network_infos.0.name";
       const podPath = "statuses.0.container_status.network_infos.0.name";
 
       return (
-        Util.findNestedPropertyInObject(task, appPath) === overlayName ||
-        Util.findNestedPropertyInObject(task, podPath) === overlayName
+        Util.findNestedPropertyInObject(task, "state") === "TASK_RUNNING" &&
```

Comment:
Would using something like `PodContainerState.RUNNING` make sense here? Read:

```diff
- Util.findNestedPropertyInObject(task, "state") === "TASK_RUNNING" &&
+ Util.findNestedPropertyInObject(task, "state") === PodContainerState.RUNNING &&
```

Reply:
Ha the same idea, but I searched for "TASK_RUNNING" in our codebase and we are using it as a string everywhere => sounds more like a refactoring in another PR.

Reply:
Great idea, let's do it once this PR is merged 👍